tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```
@@ -135,3 +135,19 @@ spec:
           # value expected by Tekton Pipelines. NOTE: If params.SSL_CERT_DIR is empty, the value
           # will contain a trailing ":" - this is ok.
           value: "/tekton-custom-certs:/etc/ssl/certs:/etc/pki/tls/certs:/system/etc/security/cacerts:$(params.SSL_CERT_DIR)"
+    - name: summary
+      image: quay.io/hacbs-contract/ec-cli:snapshot
+      command: [jq]
+      args:
+        - "."
+        - "$(results.HACBS_TEST_OUTPUT.path)"
+    - name: assert
+      image: quay.io/hacbs-contract/ec-cli:snapshot
+      command: [jq]
+      args:
+        - "--argjson"
+        - "strict"
+        - "$(params.STRICT)"
+        - "-e"
+        - ".result == "SUCCESS" or ($strict | not)"
```

I think we need this to make the YAML valid:

```suggestion
        - >
          ".result == "SUCCESS" or ($strict | not)"
```

---

alternatively these also work: 

`- '.result == "SUCCESS" or ($strict | not)'`
`- .result == "SUCCESS" or ($strict | not)`

pick your poison!